demo/d2l-filter-dropdown/d2l-demo-filter.js

```diff
@@ -0,0 +1,204 @@
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
+import '../../components/d2l-filter-dropdown/d2l-filter-dropdown.js';
+import '../../components/d2l-filter-dropdown/d2l-filter-dropdown-category.js';
+import 'd2l-menu/d2l-menu-item-checkbox.js';
+
+/**
+ * @customElement
+ * @polymer
+ */
+
+class D2LDemoFilter extends PolymerElement {
+	static get template() {
+		return html`
+			<style>
```

Comment:
Not sure if this should be `display: block` or `inline-block`? You should only need to specify the `[hidden]` CSS if the `display` varies from the default of `inline`.